shared/components/modals/WithdrawModal/WithdrawModal.js

```diff
@@ -91,8 +95,22 @@ export default class WithdrawModal extends React.Component {
     return amount
   }
 
+  getMinAmountForEthToken = () => {
+    const { data: { currency } } = this.props
+    const { currentDecimals } = this.state
+
+    let ethTokenMinAmount = '0.'
+
+    for (let a = 0; a < currentDecimals; a++) {
```

Comment:
```suggestion
    for (let a = 0; a < currentDecimals - 1; a++) {
```

Иначе на 1 разряд больше накидывает